### ISSUE TITLE
Animation sequences dont import properly

### ISSUE THREAD

**Issue by MekBoss** [niftools/blender_nif_plugin#293]
I didnt found any solution or mentioning about it in documentation.

I try import nif file with attached animation in blender 249.
It works okay, but blender plays only last animation sequence defined in kf file. Using Nifskope tool for opening kf file I can clearly see that there are several named animation sequences from which import script picks only last one. There are I can check that this is valid animations.
Also animation sequences named there as "NiControllerSequence".
I try import bones only, and play a little with import settings, but result was same.

For example, animations sequence defined in this order "stand" "attack" "death" and import script pick only "death" sequence, overriding/ignoring all previous.
Here model example in archive. Version 10.0.1.0, game "Loki: Heroes of Mythology"
[bird_is_a_word.zip](https://github.com/niftools/blender_nif_plugin/files/3410764/bird_is_a_word.zip)

Is this okay or I miss something? Or project aimed only on bethesda games support? Or I must manually edit kf file?

**Comment by MekBoss** [comment]
As working, but hacky, solution. 
Open key frame file with Nifskope, select some animation sequence, and using Ctrl + <Key Down> move it to the end of a animations list.
After that go to top menu to "Spells -> Sanitize -> Reorder Blocks" and save modified file somewhere.
If you then use this modified file as animation base for importing, it return model with animation you move to the end of the list.

**Comment by jmquigs** [comment]
The workaround fix sounds very similar to a problem I ran into recently when working the code Fallout 3/New Vegas export.  I did a fix to reorder the shapes to be near the beginning of the export list, which is something than Sanitize->Reorder appears to do.  If desired, I can make a pull request for just this change, but, it might be best to gate it behind a UI flag so that it can be toggled on/off by the users.  I can make that change as well if @neomonkeus does want this fix.

https://github.com/jmquigs/blender_nif_plugin/commit/ed052a2ea59feb05a255418f1740c22686656127
